src/Relewise.Client.Extensions/RelewiseClientOptions.cs

```diff
@@ -7,13 +7,16 @@ namespace Relewise.Client.Extensions;
 /// </summary>
 public class RelewiseClientOptions : IEquatable<RelewiseClientOptions>
 {
+    private const string BaseProductionServerUrl = "https://api.relewise.com";
```

Comment:
Ahh jeg er godt nok ikke vild med at vi angiver denne.
Som jeg skrev, så lad vær med at sætte den når du laver en Tracker, Recommender osv i stedet for at vi hardcoder denne alle mulige steder

Reply:
Jeg har undladt denne værdi nu! 🙂